tests/readme_examples_noexceptions.cpp

```diff
@@ -35,87 +35,72 @@ void basics_error_3() {
   dom::parser parser;
   dom::array cars;
   simdjson::error_code error;
-  parser.parse(cars_json).get<dom::array>().tie(cars, error);
-  if (error) { cerr << error << endl; exit(1); }
+  if (!parser.parse(cars_json).get(cars, error)) { cerr << error << endl; exit(1); }
 
   // Iterating through an array of objects
   for (dom::element car_element : cars) {
-    dom::object car;
-    car_element.get<dom::object>().tie(car, error);
-    if (error) { cerr << error << endl; exit(1); }
-
-    // Accessing a field by name
-    dom::element make, model;
-    car["make"].tie(make, error);
-    if (error) { cerr << error << endl; exit(1); }
-    car["model"].tie(model, error);
-    if (error) { cerr << error << endl; exit(1); }
-    cout << "Make/Model: " << make << "/" << model << endl;
-
-    // Casting a JSON element to an integer
-    uint64_t year;
-    car["year"].get<uint64_t>().tie(year, error);
-    if (error) { cerr << error << endl; exit(1); }
-    cout << "- This car is " << 2020 - year << "years old." << endl;
-
-    // Iterating through an array of floats
-    double total_tire_pressure = 0;
-    dom::array tire_pressure_array;
-    car["tire_pressure"].get<dom::array>().tie(tire_pressure_array, error);
-    if (error) { cerr << error << endl; exit(1); }
-    for (dom::element tire_pressure_element : tire_pressure_array) {
-      double tire_pressure;
-      tire_pressure_element.get<double>().tie(tire_pressure, error);
-      if (error) { cerr << error << endl; exit(1); }
-      total_tire_pressure += tire_pressure;
-    }
-    cout << "- Average tire pressure: " << (total_tire_pressure / 4) << endl;
-
-    // Writing out all the information about the car
-    for (auto field : car) {
-      cout << "- " << field.key << ": " << field.value << endl;
-    }
+      dom::object car;
+      if (!car_element.get(car, error)) { cerr << error << endl; exit(1); }
+
+      // Accessing a field by name
+      std::string_view make, model;
+      if (!car["make"].get(make, error)) { cerr << error << endl; exit(1); }
+      if (!car["model"].get(model, error)) { cerr << error << endl; exit(1); }
+      cout << "Make/Model: " << make << "/" << model << endl;
+
+      // Casting a JSON element to an integer
+      uint64_t year;
+      if (!car["year"].get(year, error)) { cerr << error << endl; exit(1); }
```

Comment:
My brain hurts when parsing...

```C++
if (!car["year"].get(year, error)) { cerr << error << endl; exit(1); }
```

I understand it at some level....

(By the way, in C++, you can type `not` instead of `!`.)

```C++
if (not car["year"].get(year, error)) { cerr << error << endl; exit(1); }
```

Already, to me, it looks better...

Anything to get close to...



```
if not get year from car[year]
```

but compared to English, it is not in order....